src/test/resources/omop/concept-ancestor-table-data.jsonl

```diff
@@ -1,10 +1,10 @@
-{"descendant_concept_id": 1, "ancestor_concept_id": 2}
-{"descendant_concept_id": 3, "ancestor_concept_id": 2}
-{"descendant_concept_id": 4, "ancestor_concept_id": 2}
-{"descendant_concept_id": 5, "ancestor_concept_id": 2}
-{"descendant_concept_id": 2, "ancestor_concept_id": 2}
-{"descendant_concept_id": 1, "ancestor_concept_id": 1}
-{"descendant_concept_id": 3, "ancestor_concept_id": 3}
-{"descendant_concept_id": 4, "ancestor_concept_id": 3}
-{"descendant_concept_id": 4, "ancestor_concept_id": 4}
-{"descendant_concept_id": 5, "ancestor_concept_id": 5}
+{"descendant_concept_id": 1, "ancestor_concept_id": 2, "min_levels_of_separation": 0}
```

Comment:
If we aren't going to use min levels of separation anymore, can these changes be reverted?